SECURITY.md

```diff
@@ -0,0 +1,17 @@
+# Security Policy
+https://wiki.php.net/security
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 7.3     | :heavy_check_mark: |
+| 7.2     | :heavy_check_mark: |
+| 7.1     | :heavy_check_mark: |
+| < 7.1   | :x:                |
```

Comment:
To prevent this list from getting out-dated, maybe just a link to https://www.php.net/supported-versions.php would be more appropriate.

Reply:
Okay, I replaced the table with a hyperlink. This PR also have "Allow edits from maintainers" checked.